examples/ssh/iperf.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * this driver is designed to retrieve network speed data using the iperf3.
+ * Communication protocol is SSH.
+ * This driver create a dynamic monitoring variables 
+ *      down: test for download speed.
+ *      up: test for upload speed.
+ *      UDPdown: test for UDP download speed.
+ *      UDPup: test for UDP upload speed. 
+ * 
+ * Tested with iperf3 version: v 3.7 under Ubuntu 22.04.1 LTS
+ */
+
+// Define SSH configuration
+var sshConfig = {
+    port: 22,
+    timeout: 20000
+};
+
+var _var = D.device.createVariable;
+var dload, uload, udpdload, udpuload;  // Define variables for storing download/upload speeds
+
+var UDP = "Y"; // Define whether UDP is enabled on the device (set to "Y" by default)
```

Comment:
What is this variable for?  And if its conditional why not set it a boolean? 
If its a flag to test UDP or not name it properly

eg 'shouldTestUDPSpeed'